src/docsfy/postprocess.py

```
@@ -22,8 +22,99 @@
+    """
+    lines = md_text.split("\n")
+    result: list[str] = []
+    in_fence = False
+    opening_fence_len = 0
+    opening_fence_char = "`"
+
+    for i, line in enumerate(lines):
+        stripped = line.strip()
+
+        # Track fenced code blocks (backtick and tilde fence matching)
+        if stripped.startswith("```") or stripped.startswith("~~~"):
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)